### PR TITLE
use zeekrunner scripts and zqd cli option

### PR DESCRIPTION
**Pull request by alfred-landrum**
This is part of the brimsec/brim repo side work for brimsec/brim#731. 

Use the `-zeekrunner` cli option to zqd ( https://github.com/brimsec/zq/pull/718 ), defaulting to the value of environment variable `BRIM_ZEEK_RUNNER`, else to the zeekrunner included from the zdeps zeek artifact ( https://github.com/brimsec/zeek/pull/23 ).

### CODE REVIEW

**Comment by alfred-landrum** [comment]
The zqd side of this is ready in https://github.com/brimsec/zq/pull/718 , so when this Brim PR is approved, I'll merge the zq PR, then update the zq pointers here and merge.

File: package-lock.json [hunk 1]
```diff
@@ -2258,7 +2258,7 @@
     },
     "array-equal": {
       "version": "1.0.0",
-      "resolved": "https://registry.npmjs.org/array-equal/-/array-equal-1.0.0.tgz",
+      "resolved": "http://registry.npmjs.org/array-equal/-/array-equal-1.0.0.tgz",
```
Comment (mikesbrown) [inline package-lock.json:2261]: Can you undo the https -> http here?

One way you might be able to force your npm from doing this is:
```
rm -f package-lock.json
rm -rf ~/.npm
npm install --prefer-online
git reset --hard HEAD
npm install
git diff -- package-lock.json
```
Others might have found other solutions to this and should share them if they know!

Comment (alfred-landrum) [reply package-lock.json]: I followed your instructions, and saw no change in package-lock.json, but I did fix up the entries manually.

Comment (mikesbrown) [reply package-lock.json]: Thanks for trying. Somehow I got to work. I'll keep looking into it as I can.

**Pull request merged**